### ISSUE TITLE
Hyperparameter tuning

### ISSUE THREAD

**Issue by neeraja1504** [neeraja1504/distriLLM#4]
1. Learning rate
2. epochs
3. batch size
4. optimizers (AdamW reg)
5. Change the way data is combined with positional encoding.
